phpstorm/setting_up_environment.md

```diff
@@ -3,37 +3,57 @@ title: Setting-up the Environment for PhpStorm Plugin Development
 ---
 
 ## General information
-Follow steps that are described at [Getting Started with Plugin Development](/basics/getting_started.md).
+Please review the information in the [Getting Started with Plugin Development](/basics/getting_started.md)
+section of this guide.
 
 There are two ways to develop plugins for PhpStorm:
-
-1. Use IntelliJ IDEA Ultimate with the PHP plugin installed.
-2. Use PhpStorm as a targeted IDE in a first place.
-
-The choice affects how you will configure SDK for the plugin. In the first case you need to specify current installation of IntelliJ IDEA as SDK and in the second case you need to specify current installation of PhpStorm. This step is described in [Getting Started with Plugin Development](/basics/getting_started.md).
-
-## How to use OpenAPI library
-
-> OpenAPI is available for PhpStorm 6 and above.
-
-This section explains how to configure the IntelliJ Platform for using PhpStorm OpenAPI.
-
-### Adding library to a module
-
-1. Open Project Structure **File \| Project Structure**
-2. Select **Libraries**
-3. Press **Add** button
-4. Find and select `php-openapi.jar` and `php.jar`. They are located in `<your_installation_of_PhpStorm>/plugins/php/lib`.
-
-    ![Adding Library](img/AddingLibrary.png)
-
-5. Agree to add the libraries to your Module
-6. Open **Modules \| Dependencies** and change **Scope** to **Provided**. This step is necessary because otherwise `ClassCastException` will be thrown because two instances of the library will be loaded via different class loaders
-
+1. Use IntelliJ IDEA Ultimate as the IDE with the PHP plugin installed.
+2. Use PhpStorm as the IDE.
+
+The choice affects how you will configure an SDK for developing a plugin. Using IntelliJ IDEA Ultimate as the IDE requires basing the
+SDK for your project on your installed version of IntelliJ IDEA. Using PhpStorm as the IDE requires basing the SDK for
+your project on your installed version of PhpStorm. Choosing an installed IDE as the basis for a project is described on the [Setting Up a Development Environment](/basics/getting_started/setting_up_environment.md)
+page.
+
+@anet - is IDEA Ultimate required, or could a developer use IntelliJ CE?
+
+## How to use the PhpStorm OpenAPI Library in IntelliJ
+
+This section explains how to configure the IntelliJ Platform for using the PhpStorm OpenAPI.
+
+> Note: the OpenAPI library is available for PhpStorm 6 and above.
+
+### 1. Adding the Php Library
+First the Php library must be added to your plugin's project settings.
+
+#### 1.1 Add Php Libraries to a Module
```

Comment:
This step is not needed at all (nor is 1.3). It's enough to add the library to the SDK classpath.